src/platform/datapath_epoll.c

```diff
@@ -28,6 +28,32 @@
 CXPLAT_STATIC_ASSERT((SIZEOF_STRUCT_MEMBER(QUIC_BUFFER, Length) <= sizeof(size_t)), "(sizeof(QUIC_BUFFER.Length) == sizeof(size_t) must be TRUE.");
 CXPLAT_STATIC_ASSERT((SIZEOF_STRUCT_MEMBER(QUIC_BUFFER, Buffer) == sizeof(void*)), "(sizeof(QUIC_BUFFER.Buffer) == sizeof(void*) must be TRUE.");
 
+#ifdef HAS_SENDMMSG
+#define CXPLAT_SENDMMSG sendmmsg
```

Comment:
Why even have our own define? Just implement sendmmsg directly if missing?

Reply:
My worry would be that if someone attempts to run a build built without sendmmsg on a build with sendmmsg, thats a potential ODR violation that would not be fun to debug. Doing it this way means we don't interfere with the symbols.

Reply:
And I know its static, but I've seen the linux loader do some _VERY_ weird things in the past. I'd rather just not override a libc symbol if it can be avoided.